internal/pkg/cli/svc_deploy.go

```diff
@@ -461,18 +466,32 @@ func (o *deploySvcOpts) deploySvc(addonsURL string) error {
 	return nil
 }
 
-func (o *deploySvcOpts) validateAppVersion(svc *manifest.LoadBalancedWebService) error {
-	var appVersion string
-	var err error
-	if aws.StringValue(svc.Alias) != "" && o.targetApp.RequiresDNSDelegation() {
-		appVersion, err = o.appVersionGetter.Version()
-		if err != nil {
-			return fmt.Errorf("get version for app %s: %w", o.appName, err)
-		}
-		diff := semver.Compare(appVersion, deploy.AliasLeastAppTemplateVersion)
-		if diff < 0 {
-			return fmt.Errorf(`enable "http.alias": the application version should be at least %s`, deploy.AliasLeastAppTemplateVersion)
-		}
+func validateAlias(svc *manifest.LoadBalancedWebService, app *config.Application, envName string, appVersionGetter versionGetter) error {
```

Comment:
nit: So that we can reuse it for RequestDrivenWebServices:
```suggestion
func validateAlias(alias string, app *config.Application, envName string, appVersionGetter versionGetter) error {
```